tests/e2e/test_fuzzing.py

```diff
@@ -41,3 +41,21 @@ def test_fuzzing(protostar, copy_fixture):
 """
         in result
     )
+
+
+@pytest.mark.usefixtures("init")
+def test_assume_and_reject(protostar, copy_fixture):
+    copy_fixture("test_assume_and_reject.cairo", "./tests")
+    result = protostar(
+        ["--no-color", "test", "--seed", "42", "tests/test_assume_and_reject.cairo"],
+        ignore_exit_code=True,
+    )
+
+    assert (
+        """
+[falsifying example]:
+a = 6
+b = 3
+""".rstrip()
+        in result
+    )
```

Comment:
IMHO using an integration test for this is enough here. E2E tests are slow, no need to add more of them unnecessarily